src/dumpsxiso/main.cpp

```diff
@@ -775,9 +872,15 @@ void ParseISO(cd::IsoReader& reader) {
     printf("\n");
 
     cd::IsoPathTable pathTable;
+		std::vector<cd::IsoPathTable::Entry> sorted;
 
     size_t numEntries = pathTable.ReadPathTable(&reader, descriptor.pathTable1Offs);
 
+		sorted = pathTable.pathTableList;
```

Comment:
Nitpick, but this sorted vector should be defined and assigned in one line, i.e.
```
std::vector<cd::IsoPathTable::Entry> sorted(pathTable.pathTableList);
```